sdk/iot/hub/samples/README.md

```diff
@@ -89,7 +89,93 @@ Send 5 telemetry messages using the IoT Hub Client with SAS key authentication.
 Send 5 telemetry messages using the IoT Hub Client with certificate authentication.
 
 ### [IoT Hub Twin (Certificates)][twin_sample]
-Use twin features such as updating reported properties, receiving the twin document, and receiving desired properties using the IoT Hub Client.
+Use twin features such as receiving the twin document, updating reported properties, and receiving desired properties using the IoT Hub Client.
```

Comment:
The calls "twin" -> "device twin". We need to be consistent in the name calling.

Use device twin features such as receiving the twin document, updating reported properties, and receiving desired properties using the Azure IoT Hub Client.
```suggestion
Use twin features such as receiving the twin document, updating reported properties, and receiving desired properties using the IoT Hub Client.
```